logs/console.go

```diff
@@ -69,10 +70,14 @@ func (c *consoleWriter) Init(jsonconfig string) error {
 }
 
 // WriteMsg write message in console.
-func (c *consoleWriter) WriteMsg(msg string, level int) error {
+func (c *consoleWriter) WriteMsg(when time.Time, msg string, level int) error {
 	if level > c.Level {
 		return nil
 	}
+
+	logTimeStr := formatLogTime(when)
```

Comment:
log package will write the now time,your log message will be like this  
2016-01-23 23:23:00 2016-01-23 23:23:00 ....